ArduPlane/Log.cpp

```diff
@@ -8,14 +8,7 @@ void Plane::Log_Write_Attitude(void)
     Vector3f targets;       // Package up the targets into a vector for commonality with Copter usage of Log_Wrote_Attitude
     targets.x = nav_roll_cd;
     targets.y = nav_pitch_cd;
-
-    if (quadplane.in_vtol_mode() || quadplane.in_assisted_flight()) {
-        // when VTOL active log the copter target yaw
-        targets.z = wrap_360_cd(quadplane.attitude_control->get_att_target_euler_cd().z);
-    } else {
-        //Plane does not have the concept of navyaw. This is a placeholder.
-        targets.z = 0;
-    }
+    targets.z = 0; //Plane does not have the concept of navyaw. This is a placeholder.
```

Comment:
The commit message for the existing code says "makes analysis of takeoffs easier"
I guess that's no longer true?

Reply:
as far as I can tell the copter target is just nonsense in Qassist, i'm not sure how it would have helped with takeoffs.

In full Q modes we overwrite that value lower down anyway.